src/diagnostics/LogPointsManager.ts

```diff
@@ -158,7 +165,7 @@ export class LogPointsManager extends vscode.Disposable {
         const debugSessionManager = this._debugSessionManagerMapping[params.vscodeDebugSessionId];
 
         if (!debugSessionManager) {
-            vscode.window.showErrorMessage(`Cannot find debug session with id ${params.vscodeDebugSessionId}`);
+            vscode.window.showErrorMessage(`The debug session associated with this file has expired. Please close this file and open it again from LOADED SCRIPTS explorer of an active logpoints session. More details can be found here - https://aka.ms/logpoints`);
```

Comment:
Great to see the hyper-link!  Can you deep link to an appropriate faq section?  e.g., http://aka.ms/logpoints#setting-up-logpoints links through to the setup section, since it looks like the forwarding will carry through the stuff after the `#` through on the redirect, so if there's a `#faq` section, or a `#errors` section, that we can send ppl there.

Reply:
Sounds good. Done.